### PR TITLE
fix(condo): DOMA-4134 fixed sort tickets by unitName

### PR DESCRIPTION
**Pull request by Alllex202**
Before: sort unitName as string
Снимок экрана 2022-09-13 в 17 01 07

After: sort unitName as number
Снимок экрана 2022-09-13 в 16 49 07

### CODE REVIEW

File: apps/condo/domains/common/utils/tables.utils.ts [hunk 1]
```diff
@@ -380,3 +382,12 @@ export const convertColumns = (
 export const getSorterMap: SorterMapType = (sorters) => {
     return Object.assign({}, ...sorters.map((sorter) => ({ [sorter.columnKey]: sorter.order })))
 }
+
+export const sorterUnitName: (a: Ticket, b: Ticket) => number = (a: Ticket, b: Ticket) => {
```
Comment (akarjakin) [inline apps/condo/domains/common/utils/tables.utils.ts:386]: bad function name. `sortUnitName`

File: apps/condo/domains/common/utils/tables.utils.ts [hunk 2]
```diff
@@ -380,3 +382,12 @@ export const convertColumns = (
...
+    const firstUnitName = get(a, 'unitName', null)
+    const secondUnitName = get(b, 'unitName', null)
+    if (!firstUnitName && !secondUnitName) return 0
+    if (!secondUnitName) return -1
+    if (!firstUnitName) return 1
+    return Number(firstUnitName) - Number(secondUnitName)
```
Comment (akarjakin) [inline apps/condo/domains/common/utils/tables.utils.ts:392]: there could be case with NaN, I think you should handle it too. And please add tests for this sorter function.

**Review by akarjakin — CHANGES_REQUESTED** [review]

File: apps/condo/domains/common/utils/tables.utils.ts (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/common/utils/tables.utils.ts:392]: first 3 conditions also check NaN

File: apps/condo/domains/common/utils/tables.utils.ts (hunk 1 above)
Comment (sitozzz) [inline apps/condo/domains/common/utils/tables.utils.ts:386]: Please add tests for this util function

Comment (AleX83Xpert) [reply apps/condo/domains/common/utils/tables.utils.ts:386]: This is actually a sorter.
`unitNameSorter`

**Review by AleX83Xpert — APPROVED** [review]

**Comment by Alllex202** [comment]
unitName can be a string, not just a number. And the sorting happens on the server, not on the client. It's better to leave it as is.

**Pull request closed without merging**